extensions/amp-facebook/1.0/test-e2e/test-bento-facebook.js

```diff
@@ -0,0 +1,29 @@
+describes.endtoend(
+  'bento-facebook',
+  {
+    version: '1.0',
+    fixture: 'bento/facebook.html',
+    environments: ['single'],
```

Comment:
not too familiar with the e2e environments, but why use `'single'` instead of the default `'ampdoc-preset'`?

Reply:
From this README, https://github.com/ampproject/amphtml/blob/main/build-system/tasks/e2e/README.md#writing-e2e-tests

I was under the impression that there are only 3 types of environments.

> `// environments: ['single', 'viewer-demo', 'shadow-demo']`

But looking at https://github.com/ampproject/amphtml/blob/main/build-system/tasks/e2e/describes-e2e.js#L256

I assume that `ampdoc-preset` is all 3 of them?

Anyway, viewer-demo and shadow-demo don't work correctly in web component stand alone mode. And I wasn't able to figure out why, so I just cut out those two environments.